### DISCUSSION TITLE
Support backticks formatting on PR/Issue titles

### DISCUSSION THREAD

**Discussion by rwp0** [community/community#72781] [Mobile]
Currently GitHub web supports it when we use backticks to format PR or issue titles, but the mobile version still has the backticks visible with the text remaining unformatted.

I'll attach screenshots as examples:

### GH Web PR's
![Screenshot_20231028_134521_Vivaldi Browser Snapshot.jpg](https://github.com/community/community/assets/68062695/ccaa83c5-0e11-4b68-9750-a7a88e78a384)

### GH Mobile PR's
![Screenshot_20231028_134115_GitHub.jpg](https://github.com/community/community/assets/68062695/bdcaaf5c-6c05-48f5-9dc2-b2451ffd51ce)

### GH Mobile Issues
![Screenshot_20231028_134127_GitHub.jpg](https://github.com/community/community/assets/68062695/5d261d2e-8432-403a-8348-acdac9ea037d)

**Comment by Iron-Ham** [comment]
Hi @rwp0 
Thank you for the feedback. I notice that this issue seems to be specific to Android. I'll open up an internal issue to track this, so that we can support backticks in issue and pull request titles on all platforms. 


Thanks,
Hesham

**Reply by rwp0** [reply]
How do we track those "internal tickets"?
Through this same ticket?
Thanks

**Comment by kauaneiras** [comment]
🚀
